homeassistant/components/velbus/config_flow.py

```diff
@@ -82,3 +85,40 @@ async def async_step_user(
             ),
             errors=self._errors,
         )
+
+    async def async_step_usb(self, discovery_info: usb.UsbServiceInfo) -> FlowResult:
+        """Handle USB Discovery."""
+        vid = discovery_info.vid
+        pid = discovery_info.pid
+        serial_number = discovery_info.serial_number
+        device = discovery_info.device
+        manufacturer = discovery_info.manufacturer
+        description = discovery_info.description
+        await self.async_set_unique_id(
+            f"{vid}:{pid}_{serial_number}_{manufacturer}_{description}"
+        )
```

Comment:
I'm a bit confused about this unique_id, we don't set one in the user flow?

So, how does this work/help?

Reply:
this unique id does not seem to be needed, so its removed

Reply:
This is actually not working correctly not and this PR has been merged without this being addressed (@bdraco).

So, the unique ID is not just for ignoring (which yes, is required), but also for detecting if something is already set up. For example; setting up an integration that has been previously ignored.

The unique ID should be consistently used in the integration; Not at will.

Please adjust/fix this behavior in a follow-up PR.

Reply:
Sorry I missed that.

I figured this check would prevent duplicates 
```python3
        if self._prt_in_configuration_exists(dev_path):
            return self.async_abort(reason="already_configured")
```

Looking again it is not good enough though since the same port could be setup with a different path since linux allows multiple device entries for the same device.